src/components/m-header-mobile-others/index.js

```diff
@@ -17,7 +17,9 @@ class AXAHeaderMobileOthers extends BaseComponentGlobal {
   }
 
   init() {
-    super.init({ styles, template });
+    super.init({
```

Comment:
Well, generally we should allow object literal one liners or `eslint --fix` is broken